E.Deezer/E.Deezer.csproj

```diff
@@ -1,103 +1,23 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <MinimumVisualStudioVersion>10.0</MinimumVisualStudioVersion>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{7DC0FD2E-3A4F-49A2-94F1-E54D51835948}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>E.Deezer</RootNamespace>
-    <AssemblyName>E.Deezer</AssemblyName>
-    <DefaultLanguage>en-GB</DefaultLanguage>
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <TargetFrameworkProfile>Profile111</TargetFrameworkProfile>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <ShouldCreateLogs>True</ShouldCreateLogs>
-    <AdvancedSettingsExpanded>False</AdvancedSettingsExpanded>
-    <UpdateAssemblyVersion>True</UpdateAssemblyVersion>
-    <UpdateAssemblyFileVersion>True</UpdateAssemblyFileVersion>
-    <UpdateAssemblyInfoVersion>True</UpdateAssemblyInfoVersion>
-    <AssemblyVersionSettings>None.None.None.Increment</AssemblyVersionSettings>
-    <AssemblyFileVersionSettings>None.None.None.Increment</AssemblyFileVersionSettings>
-    <AssemblyInfoVersionSettings>DayStamp.MonthStamp.YearStamp.TimeStamp</AssemblyInfoVersionSettings>
-    <AssemblyInfoFilePath>Properties\AssemblyInfo.cs</AssemblyInfoFilePath>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageId>E.Deezer</PackageId>
+    <Version>3.0.0</Version>
+    <Authors>E</Authors>
+    <Product />
+    <PackageProjectUrl>https://github.com/projectgoav/E.Deezer</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/projectgoav/E.Deezer</RepositoryUrl>
+    <PackageTags>.NET Rest Deezer API</PackageTags>
+    <Copyright>Copyright 2017</Copyright>
+    <PackageReleaseNotes>Summary of changes made in this release of the package.</PackageReleaseNotes>
+    <Description>.NET Deezer API wrapper</Description>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Can we have `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` here so we can use the AssemblyInfo.cs file?